src/App.jsx

```diff
@@ -9,11 +9,15 @@ import { useAuth } from './contexts/AuthContextProvider';
 import { db } from './utils/FirebaseConfig';
 import { doc, getDoc, setDoc } from 'firebase/firestore';
 
+import { useHistory } from './utils/useHistory';
+
 const App = () => {
 	const { user } = useAuth();
 
 	let navigate = useNavigate();
 
+  useHistory();
```

Comment:
Formatting? Graag prettier/eslint in orde brengen

Reply:
Ik krijg hier opnieuw geen errors... Heb opgeslaan dus zou moeten gefixt zijn. Ik veronderstel dat het de uitlijning was?

Reply:
Idd de uitlijning. Deze lijkt nog altijd niet in orde...

Reply:
Heb je in VSCode Prettier en EsLint plugin geinstalleerd?

Reply:
Yep, ze staan allebei geïnstalleerd en actief...

Reply:
Yep, ze staan allebei geïnstalleerd en actief...